src/display.hpp

```diff
@@ -110,7 +110,7 @@ class display : public gui2::top_level_drawable
 
 	/** The viewing team is the team currently viewing the game. */
 	std::size_t viewing_team() const { return currentTeam_; }
-	int viewing_side() const { return currentTeam_ + 1; }
+	int viewing_side() const { return static_cast<int>(currentTeam_) + 1; }
```

Comment:
Seems like fixing the wrong problem, `viewing_team` and `viewing_side` should return the same type.

This one's in the third commit of the branch, so sorry for not catching it earlier.

Reply:
Thanks for the explanation. Not marking as "resolved" yet, because it's a reminder to work out where to improve the documentation.

Reply:
Then we should remove `viewing_team()` entirely.

Reply:
Seconded.